chapters/fa/chapter2/2.mdx

```diff
@@ -75,15 +75,15 @@ classifier(
 
 ## پیش‌پردازش با توکِنایزر
 
-مثل شبکه‌های عصبی دیگر، مدل‌های ترنسفورمر هم نمی‌توانند نوشته خام را پردازش کنند. پس اولین قدم در خط تولید ما تبدیل نوشته خام ورودی به اعدادی است که مدل می‌فهمد. برای این کار از یک *توکِنایزر* استفاده می‌کنیم، که مسئولیت‌های زیر را بر عهده دارد:
+مثل شبکه‌های عصبی دیگر، مدل‌های ترنسفورمر هم نمی‌توانند نوشته خام را پردازش کنند. پس اولین قدم در خط تولید ما تبدیل نوشته خام ورودی به اعدادی که مدل بفهمد است. برای این کار از یک *توکِنایزر* استفاده می‌کنیم، که مسئولیت‌های زیر را بر عهده دارد:
```

Comment:
suggestion:
"پس اولین قدم در خط تولید ما، تبدیل نوشته خام ورودی به اعدادی است که مدل قادر به فهم آن‌ها باشد."
instead of:
"پس اولین قدم در خط تولید ما تبدیل نوشته خام ورودی به اعدادی که مدل بفهمد است"

Reply:
I like this suggestion, applied.

No half-space in آنها per the style guide.